type_bytearray.go

```diff
@@ -292,12 +292,21 @@ func (b *byteArrayDeltaEncoder) Close() error {
 }
 
 type byteArrayStore struct {
-	repTyp   parquet.FieldRepetitionType
-	min, max []byte
+	repTyp parquet.FieldRepetitionType
+	st     stats
+	pagest stats
```

Comment:
pageSt? or stPage? or call 'stats' 'statistics' instead and the struct fields stats & pageStats?

Reply:
I renamed several identifiers to make things clearer.